Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// Added a test for Empty Input.
+// Created branch 'review' for PR sharing.
+
+
+namespace PalindromeArray.UnitTests
+{
+    [TestClass]
+    public class PalindromeUnitTests
+    {
+        [TestMethod]
+        public void Check_Empty_Input()
+        {
+            // Arrange
+            Palindrome check = new Palindrome();
+            char[] arrToCheck = check.PalindromeHelper("");
+            // Act
+            bool boolResultOfPalindromeCheck = check.IsArrayPalindrome(arrToCheck);
+            // Assert
+            Assert.AreEqual(false, boolResultOfPalindromeCheck);
+        }
+        [TestMethod]
+        public void Check_Longest_Palindrome_In_Dictionary()
+        {
+            Palindrome check = new Palindrome();
+            // The longest palindromic word in the Oxford English Dictionary is "tattarrattat", 
+            // coined by James Joyce in Ulysses (1922) for a knock on the door.
+            char[] arrToCheck = check.PalindromeHelper("tattarrattat");
+
+            bool boolResultOfPalindromeCheck = check.IsArrayPalindrome(arrToCheck);
+
+            Assert.AreEqual(true, boolResultOfPalindromeCheck);
+        }
+        [TestMethod]
+        public void Check_Odd_Palindrome_One()
+        {
+            Palindrome check = new Palindrome();
+            char[] arrToCheck = check.PalindromeHelper("radar");
+
+            bool boolResultOfPalindromeCheck = check.IsArrayPalindrome(arrToCheck);
+
+            Assert.AreEqual(true, boolResultOfPalindromeCheck);
+        }
+        [TestMethod]
+        public void Check_Odd_Palindrome_Two()
```

Comment:
There's no need for two different tests since they are testing the exact same scenario (true for both even and odd).

Reply:
Will remove redundant test cases and useless code.